### PR TITLE
Added Scroll Back To Top

### PR DESCRIPTION
**Pull request by Yoncity**
#### What does this PR do?
This PR adds a button to scroll to the top of the page.

#### Description of Task to be completed?
Add a button to scroll to the top
If at the top, hide the button

#### How should this be manually tested?
Fetch and checkout to `ft-scroll-to-top`
Run the server with an active internet connection
Scroll down and the button to `Scroll To Top` will appear
Click the button and will it you to the top and dissappear.

#### Screenshots (if appropriate)
Screenshot from 2020-04-26 14-42-41
Screenshot from 2020-04-26 14-42-48

### CODE REVIEW

**Comment by gracelungu** [comment]
@Yoncity Do you mind reducing the size of the button in half the current size? it looks imposing.

**Comment by gracelungu** [comment]
@Yoncity Resolve the conflict, be careful while accepting changes.

**Review by oesukam — CHANGES_REQUESTED** [review]
You can reduce a little bit on the shadow as we are pretty much not using shadows

**Comment by Yoncity** [comment]
> You can reduce a little bit on the shadow as we are pretty much not using shadows

Reduced it to 1px

**Comment by Yoncity** [comment]
> @Yoncity Resolve the conflict, be careful while accepting changes.

Reduced to 20 x 20px

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -1,32 +1,61 @@
...
+    <Layout {...props}>
+      <div className="container-fluid">
+        <div className="row home-page">
+          <div className="col-md-3">
+            <HomeLeft {...props} />
+          </div>
+          <div className="col-md-6">
+            <Post />
+            <Feed {...props} />
+            <button
```
Comment (oesukam) [inline src/pages/Home/Home.jsx]: What about other pages?

File: src/components/ScrollToTop/ScrollToTop.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    &:hover {
+      box-shadow: 0px 0px 1px #aaa;
+      transition: all 0.2s linear;
+    }
+
+    &.hide-back-to_top {
+      display: none;
+    }
+  }
```
Comment (oesukam) [inline src/components/ScrollToTop/ScrollToTop.scss]: Add new line at the end of the file

**Review by oesukam — CHANGES_REQUESTED** [review]

File: src/components/ScrollToTop/ScrollToTop.scss (hunk 2 above)
Comment (Yoncity) [reply src/components/ScrollToTop/ScrollToTop.scss]: Done.

**Review by oesukam — APPROVED** [review]
LGTM

**Pull request merged**